okhttp/third_party/okhttp/test/java/io/grpc/okhttp/internal/framed/HpackTest.java

```diff
@@ -49,7 +49,7 @@
 
   @Before public void reset() {
     hpackReader = newReader(bytesIn);
-    hpackWriter = new Hpack.Writer(bytesOut);
+    hpackWriter = new Hpack.Writer(4096, false, bytesOut);
```

Comment:
I would do this only for new tests that need it.

Reply:
Previously the default is to not use compression and no dynamic table (i.e., not table size setting), but now we have these definitions, even for existing tests that do not rely on them.